.gitignore

```diff
@@ -55,19 +55,20 @@ docs/errors.txt
 docs/images/auto-generated
 **/myscreenshot.png
 
-# flycheck temp files
-flycheck*
-
 
 docs/external_examples.rst
 
 # hypothesis
 .hypothesis
 
+# emacs (> vim)
```

Comment:
[citation needed]

Reply:
Haha!  Glad someone noticed that.  Figured I keep that in until someone noticed.  It took 15 minutes.

Reply:
It was a nice try ;)